chrome/content/zotero/xpcom/zotero.js

```diff
@@ -1142,7 +1137,25 @@ Services.scriptloader.loadSubScript("resource://zotero/polyfill.js");
 				nsIEPS.loadUrl(uri);
 			}
 		}
-	}
+	};
+     /**
+      * Launch a file with the given application
+      */
+     this.launchFileWithGivenApplication = function (applicationPath, file) {
+         var exec = Components.classes["@mozilla.org/file/local;1"]
+             .createInstance(Components.interfaces.nsILocalFile);
+         exec.initWithPath(applicationPath);
+         if (!exec.exists()) {
+             throw new Error("'" + applicationPath + "' does not exist");
+         }
+
+         var proc = Components.classes["@mozilla.org/process/util;1"]
+             .createInstance(Components.interfaces.nsIProcess);
+         proc.init(exec);
+
+         var args = [file.path];
```

Comment:
Just `file`, once the calls above are fixed.

Reply:
I think we still need the file.path as an argument here, won't we? I mean from commandline, i would execute it like this:
´´<<path to pdf reader>> <<path to file>>´´

if i would use the use file as an argument instead of file.path i would try to pass a file to the commandline execution. The way i understand the documentation of nsIProcess that wouldn't work as intendet, would it?